src/map/battle.cpp

```diff
@@ -3632,7 +3632,8 @@ static void battle_calc_skill_base_damage(struct Damage* wd, struct block_list *
 				}
 #endif
 				if(sd->status.party_id && (skill=pc_checkskill(sd,TK_POWER)) > 0) {
-					if( (i = party_foreachsamemap(party_sub_count, sd, 0)) > 1 ) { // exclude the player himself [Inkfish]
+					i = party_foreachsamemap(party_sub_count, sd, 0) - 1; // Bonus Attack multiplier based on party members count doesn't include own player [Tydus1] [Inkfish]
```

Comment:
This actually has potential to break. If somehow **i** was ever equal 0, which normally wouldn't happen in this case, but you never know! You would be subtracting 1 from 0 making it -1. Under normal circumstances wouldn't be a big deal for the below if statement. However this case is different because we're returning an int from **party_foreachsamemap** to a **uint16 i**, which makes it unsigned, therefore it would underflow and equal 65535 instead of -1. I want to recommend the following change so this underflow can't possibly happen and would always cap at 0 before going negative:

`i = max(party_foreachsamemap(party_sub_count, sd, 0) - 1, 0); // Bonus Attack multiplier based on party members count doesn't include own player [Tydus1] [Inkfish]`

Reply:
Done! Thanks for the review @NeutralDev